File: python/character_match.py
# Don't forget to run your tests!

def is_character_match(string1, string2):
	# Your code here
	res = True 
	string1 = string1.lower()
	string2 = string2.lower()
	for i in string1:
		if i in string2:
			res = True
		else: return False
	return res
			


# print(is_character_match('charm', 'march'))

File: js/characterMatch.js
exports.isCharacterMatch = function (string1, string2) {
  string1 = string1.toLowerCase();
  string2 = string2.toLowerCase();
  for (let i = 0; i < string1.length; i++) {
    if (string2.includes(string1[i])) {
      return true;
    } else {
      return false;
    }
  }
};
